packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -521,3 +525,27 @@ gmaps.LatLng _pixelToLatLng(gmaps.GMap map, int x, int y) {
 
   return projection.fromPointToLatLng!(point)!;
 }
+
+String _sanitizeHtml(String htmlString) {
```

Comment:
This is a best effort attempt, since Firefox and Safari will fail to create a policy.

Reply:
Yes, this is unfortunately the case. I forgot linking this before, but here's what I did the last time I did something like this:

* https://github.com/flutter/packages/blob/main/packages/google_identity_services_web/lib/src/js_loader.dart#L27-L53

And here's the tweaks to add the special getters/setters to make things slightly easier:

* https://github.com/flutter/packages/blob/main/packages/google_identity_services_web/lib/src/js_interop/package_web_tweaks.dart (for example: making window.trustedTypes nullable, or setting a trustedType as the src of a Script element)

Reply:
Yeah, I recognize that pattern, I had to do something similar in a different package for yet another API that wasn't supported. I will add this adjustment as well.